lib/tests/suites/resource_test.rb

```diff
@@ -675,9 +675,18 @@ def teardown
           result.update(STATUS[:skip],"Preexisting #{resource_class.name.demodulize} unknown.", nil)
         else
           reply = @client.destroy(@resource_class,@preexisting_id)
-          if reply.code==204
+          if reply.code==204 and reply.body.blank?
```

Comment:
There were a few of these... I didn't mark them all, so make sure you change all of them.